integration/gcp_generation_test.go

```diff
@@ -15,6 +15,51 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const (
+	organizationId = "org-1"
+	projectId      = "project-1"
+)
+
+type MsgRsp struct {
+	message      string
+	response     string
+	skipResponse bool
+	menuItems    int
+}
+
+func msgRsp(message string, response string) MsgRsp {
+	return MsgRsp{message, response, false, 0}
+}
+
+func msgOnly(message string) MsgRsp {
+	return MsgRsp{message, "", true, 0}
+}
+
+func msgMenu(message string, count int) MsgRsp {
+	return MsgRsp{message, "", true, count}
+}
+
+func cliExpectations(t *testing.T, c *expect.Console, m []MsgRsp) {
+	for _, e := range m {
+		expectString(t, c, e.message)
+
+		if !e.skipResponse {
+			c.SendLine(e.response)
+		}
+
+		if e.menuItems > 0 {
+			for i := 0; i < e.menuItems-1; i++ {
+				c.Send("\x1B[B")
+			}
+			c.SendLine("\x1B[B")
+		}
+	}
+}
```

Comment:
NICE!

Reply:
Really cleans up the test cases.  Ideally this should be using command pattern and interfaces.  Something to improve/tidy, along with putting in lib/utility module, if/when applied to azure and aws integration tests.

Reply:
I wonder if this could be another candidate to move to a helper file? CC: @dmurray-lacework

Reply:
We have framework_test.go for so reusable testing funcs

Reply:
There's an expectsString, perhaps rename this to expectsCliOutput, or something like that